include/aws/auth/credentials.h

```diff
@@ -407,18 +407,18 @@ struct aws_credentials_provider_chain_default_options {
 };
 
 /**
- * For language binding to provide credentials from that language
+ * Configuration options for the default credentials provider chain.
```

Comment:
delegate, and in this case it's not a chain, just a single provider.

Reply:
whoops... copy/paste error again...